Misc VLM pipeline doc updates (#523)
NVIDIA-AI-Blueprints/rag: docs/change-model.md

### Configure Embedding Dimensions

> The default embedding model (`nvidia/llama-nemotron-embed-vl-1b-v2`) uses **2048 dimensions** by default. When changing to a different embedding model, you may need to update the dimensions to match the model's output.

Reviewer: ```suggestion
The default embedding model (`nvidia/llama-nemotron-embed-vl-1b-v2`) uses 2048 dimensions by default. When changing to a different embedding model, you may need to update the dimensions to match the model's output.
```

## Switch from the VLM Embedder to the Text-Only Embedder

> The default embedder is the multimodal `nvidia/llama-nemotron-embed-vl-1b-v2` (2048 dimensions), which embeds both text passages and page images. To revert to the text-only `nvidia/llama-nemotron-embed-1b-v2` (also 2048 dimensions, same vector space size, smaller GPU footprint), follow the steps below. Ingestion and retrieval must use the **same** embedding model — re-ingest your documents after switching.

Reviewer: ```suggestion
The default embedder is the multimodal `nvidia/llama-nemotron-embed-vl-1b-v2` (2048 dimensions), which embeds both text passages and page images. To revert to the text-only `nvidia/llama-nemotron-embed-1b-v2` (also 2048 dimensions, same vector space size, smaller GPU footprint), follow the steps below. Ingestion and retrieval must use the same embedding model — re-ingest your documents after switching.
```

1. Override via env var (simplest):

   ```bash
   export APP_EMBEDDINGS_MODELNAME="nvidia/llama-nemotron-embed-1b-v2"
   ```

2. Restart the rag and ingestor servers:

   ```bash
   docker compose -f deploy/compose/docker-compose-ingestor-server.yaml up -d
   docker compose -f deploy/compose/docker-compose-rag-server.yaml up -d
   ```

The text-only embedding NIM (`nemotron-embedding-ms` in [`deploy/compose/nims.yaml`](../deploy/compose/nims.yaml)) is started by the same profile that the VLM embedder uses, so no separate NIM startup is needed.

### Helm

In [`values.yaml`](../deploy/helm/nvidia-blueprint-rag/values.yaml), set the model name in the rag-server and ingestor-server env-var blocks (search for the existing `llama-nemotron-embed-vl-1b-v2` references):

```yaml
envVars:
  APP_EMBEDDINGS_MODELNAME: "nvidia/llama-nemotron-embed-1b-v2"

ingestor-server:
  envVars:
    APP_EMBEDDINGS_MODELNAME: "nvidia/llama-nemotron-embed-1b-v2"
```

The text-only embedding NIM (`nvidia-nim-llama-32-nv-embedqa-1b-v2` under `nimOperator`) is already configured in `values.yaml`; no image swap is required. Apply with [Change a Deployment](deploy-helm.md#change-a-deployment).

:::{warning}

> **Re-ingest after switching.** Vectors produced by the VLM embedder are not directly comparable to vectors from the text-only embedder; retrieval accuracy will degrade until you re-ingest your corpus.

Reviewer: ```suggestion
Re-ingest after switching. Vectors produced by the VLM embedder are not directly comparable to vectors from the text-only embedder; retrieval accuracy will degrade until you re-ingest your corpus.
```

:::

## Switch to the VLM Reranker

> The default reranker is the text reranker `nvidia/llama-nemotron-rerank-1b-v2`. To use a multimodal reranker that can re-rank with awareness of cited images, switch to `nvidia/llama-nemotron-rerank-vl-1b-v2`. See [Multimodal Retriever — Part 2: VLM Reranker](multimodal-retriever.md#part-2--vlm-reranker) for what the multimodal reranker does and the `ENABLE_VLM_RERANKER_IMAGE_INPUT` flag in detail. The steps below cover the model swap.

Reviewer: ```suggestion
The default reranker is the text-based model `nvidia/llama-nemotron-rerank-1b-v2`. If you need image-aware reranking, switch to the multimodal model `nvidia/llama-nemotron-rerank-vl-1b-v2`, which can incorporate cited images into its ranking decisions. For a detailed explanation of how this works, see Multimodal Retriever — Part 2: VLM Reranker and the ENABLE_VLM_RERANKER_IMAGE_INPUT flag. The steps below walk through how to swap the model.
```

> ## Switch the Text Embedding Model

Reviewer 2: Change embedding model

Author: Handled

The default text embedding model is `nvidia/llama-nemotron-embed-1b-v2` (2048 dimensions). Use this section to switch to a different text embedding NIM with both Docker Compose and Helm. Both ingestion and retrieval must use the **same** embedding model and dimensions; if you change either, re-ingest your documents.

> 

Reviewer 2: remove `text`

Author: Done

:::{tip}
For the multimodal/VLM embedder, see [VLM Embedding for Ingestion](vlm-embed.md). This section covers text-only embedding NIMs.

> :::

Reviewer 2: Why a separate doc is needed for VLM embed model change? Steps are same to switch to VLM embed/rerank model if text chunks are used

Author: Updated as discussed, created dedicated `multimodal-retriever.md`
